PATCH-P0-FIX-UPSTREAM: Fixed paths in installing languages

Maintainer: Michal Hrusecky <Michal.Hrusecky@opensuse.org>

Index: sql/share/Makefile.am
===================================================================
--- sql/share/Makefile.am.orig
+++ sql/share/Makefile.am
@@ -23,7 +23,7 @@ dist-hook:
 	  test -d $(distdir)/$$dir || mkdir $(distdir)/$$dir; \
 	  $(INSTALL_DATA) $(srcdir)/$$dir/*.* $(distdir)/$$dir; \
 	done; \
-	sleep 1 ; touch $(srcdir)/*/errmsg.sys
+	sleep 1 ; touch $(builddir)/*/errmsg.sys
 	$(INSTALL_DATA) $(srcdir)/charsets/README $(distdir)/charsets
 	$(INSTALL_DATA) $(srcdir)/charsets/Index.xml $(distdir)/charsets
 
@@ -40,7 +40,7 @@ install-data-local:
 	for lang in @AVAILABLE_LANGUAGES@; \
 	do \
 		$(mkinstalldirs) $(DESTDIR)$(pkgdatadir)/$$lang; \
-		$(INSTALL_DATA) $(srcdir)/$$lang/errmsg.sys \
+		$(INSTALL_DATA) $(builddir)/$$lang/errmsg.sys \
 			        $(DESTDIR)$(pkgdatadir)/$$lang/errmsg.sys; \
 	done
 	$(mkinstalldirs) $(DESTDIR)$(pkgdatadir)/charsets
